files/es/web/html/element/caption/index.md

```diff
@@ -0,0 +1,261 @@
+          >Declaraciones de estilo</a
+        >.
+      </td>
+      <td>Lo fija el navegador.</td>
+    </tr>
+    <tr>
+      <td>
+        <a
+          class="external"
+          href="http://html.conclase.net/w3c/html401-es/struct/dirlang.html#adef-lang"
+          ><span style="color: green">lang</span></a
+        >
+      </td>
+      <td>
+        Información sobre el
+        <a
+          class="external"
+          href="http://html.conclase.net/w3c/html401-es/struct/dirlang.html"
+          >idioma del contenido</a
+        >
+        del elemento y del valor de sus atributos.
+      </td>
+      <td>
+        Un
+        <a
+          class="external"
+          href="http://html.conclase.net/w3c/html401-es/types.html#type-langcode"
+          >código de idioma</a
+        >.
+      </td>
+      <td>
+        Lo fija el navegador. <abbr title="Por defecto">PD.</abbr>:
+        "desconocido".
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <a
+          class="external"
+          href="http://html.conclase.net/w3c/html401-es/struct/dirlang.html#adef-dir"
+          ><span style="color: green">dir</span></a
+        >
+      </td>
+      <td>
+        Indica la
+        <a
+          class="external"
+          href="http://html.conclase.net/w3c/html401-es/struct/dirlang.html"
+          >dirección de texto</a
+        >
+        y tablas.
+      </td>
+      <td>
+        Uno de los siguientes: <abbr title="Left-to-right">'ltr' </abbr>o
+        <abbr title="Right-to-left">'rtl'.</abbr>
+      </td>
+      <td>
+        Lo fija el navegador. <abbr title="Por defecto">PD.</abbr>: En
+        castellano 'ltr'.
+      </td>
+    </tr>
+    <tr>
+      <td colspan="4">
+        <strong>Eventos</strong>:
+        <code style="color: green"
+          >onclick, ondblclick, onmousedown, onmouseup, onmouseover,
+          onmousemove, onmouseout, onkeypress, onkeydown, onkeyup.</code
+        >
+      </td>
+    </tr>
+    <tr>
+      <th colspan="4">Específicos</th>
+    </tr>
+    <tr>
+      <td colspan="4">No tiene</td>
+    </tr>
+    <tr>
+      <th colspan="4">De transición</th>
+    </tr>
+    <tr>
+      <td>
+        <a
+          class="external"
+          href="http://html.conclase.net/w3c/html401-es/struct/dirlang.html#adef-dir"
+          ><span style="color: green">align</span></a
+        >
+      </td>
+      <td>Indica la alineación en relación a la tabla.</td>
+      <td>Uno de los siguientes: 'top', 'bottom', 'left', 'right'.</td>
+      <td>Lo fija el navegador.</td>
+    </tr>
+    <tr>
+      <th>atributo</th>
+      <th>descripción</th>
+      <th>valor</th>
+      <th>por omisión</th>
+    </tr>
+  </tbody>
+</table>
+
+### Ejemplos de uso
+
+#### Ejemplo simple
+
+El siguiente código:
+
+```
+
+  <table border='1'>
+   <caption>Tabla con caption</caption>
+    <tr> <td> tabla de una celda. </td>  </tr>
+  </table>
+
+```
+
+Se visualiza así:
+
+| tabla de una celda. |
+| ------------------- |
```

Comment:
to view the example

````suggestion
```html
<table border='1'>
  <caption>Tabla con caption</caption>
  <tr> <td> tabla de una celda. </td>  </tr>
</table>
```

```css hidden
table {
  width: 100%;
}
```

Se visualiza así:

{{EmbedLiveSample}}
````